UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -29,16 +27,16 @@ import java.util.concurrent.TimeUnit
 class UniConnectUnitTest {
 
     //lateinit var mvm : MainViewModel
-
+//
     //@MockK
     //lateinit var mockUniService : UniversityService
```

Comment:
While you're here, remove the commented out code.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.